### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by vasulrgdevera** [jbcarreon123/Win11CompChk#4]
The exported report does not display CPU information. There is a question mark in the graphical interface, there are no marks in the exported report.

**Comment by jbcarreon123** [comment]
Sorry if it's late. My PC hasn't been fixed during that time frame.

Are you used the delfiles.bat and selected 3?

**Comment by vasulrgdevera** [comment]
![win11_test1](https://user-images.githubusercontent.com/62345949/148030295-5e47e2b9-ef2b-40fa-86fa-4a52a7b8350b.jpg)
![Win11_test2](https://user-images.githubusercontent.com/62345949/148030296-1e71d923-253a-412b-b619-585134ccc92d.jpg)
Yes, I press "E" Export Result, then I press "N" Normal Text, but in Result file does not symbol "?" or "X" near CPU information.

**Comment by maduesi** [comment]
Hi,

the result file does not output "OK", "X" or "?".

"OK", "X" or "?" are stored in variables like %are%, %cput%, %cre%, %crre%, %cfre% and are only output in the console
echo  %are%       Arch:          %CPUA% CPU, %OSA% OS
echo  %cput%     CPU Category:
echo  %cre%     ├─CPU:           %CPUn%
echo  %crre%     ├─CPU Core #:    %cpucn% Core(s), %cpulp% Logical Processor(s)
echo  %cfre%     └─CPU Freq:      %CPUf%MHz

**Comment by jbcarreon123** [comment]
> Hi,
> 
> the result file does not output "OK", "X" or "?".
> 
> "OK", "X" or "?" are stored in variables like %are%, %cput%, %cre%, %crre%, %cfre% and are only output in the console echo %are% Arch: %CPUA% CPU, %OSA% OS echo %cput% CPU Category: echo %cre% ├─CPU: %CPUn% echo %crre% ├─CPU Core #: %cpucn% Core(s), %cpulp% Logical Processor(s) echo %cfre% └─CPU Freq: %CPUf%MHz

You are correct. Notepad does not make the font in line. But I will add it as a feature to v6.
